flink-runtime/src/main/java/org/apache/flink/runtime/jobmaster/JobManagerServices.java

```diff
@@ -116,12 +116,17 @@ public static JobManagerServices fromConfiguration(
 		final FiniteDuration timeout;
 		try {
 			timeout = AkkaUtils.getTimeout(config);
-		} catch (NumberFormatException e) {
+		}
+		catch (NumberFormatException e) {
```

Comment:
Yes, that is still my old code style config. IntelliJ sometimes triggers some local reformatting.
@shijinkui Updating the code style has been a discussion forever. To include this into the style, one would need to fix many styles. But it is ultimately a good idea to have this, agreed.